src/Components/SmartComponents/CVEs/CVEs.test.js

```diff
@@ -240,12 +228,12 @@ describe('CVEs', () => {
 
 });
 
-describe('CVEs RTL', () => {
+describe('CVEs default filters are enabled, no other filters are added in the URL', () => {
     it('Should show extended Systems filter when edge parity feature is disabled', async () => {
         render(
-            <TestWrapper store={ store }>
+            <ComponentWithContext store={ store } renderOptions={{ initialEntries: ['?affecting=rpmdnf%2Cedge&advisory_available=true']}}>
                 <CVEs />
-            </TestWrapper>
+            </ComponentWithContext>
         );
```

Comment:
It feels like TestWrapper and ComponentWithContext both deal with the same problem: render a component with the necessary providers and contexts. We have to stop duplicating these two components and just stick to only one. I see that TestWrapper doesn't support initialEntries: let's perhaps extend it and keep the usage of it here in this test file?

Reply:
I would remove TestWrapper. Because `ComponentWithContext` follows the conventions that we have in other [apps](https://github.com/RedHatInsights/insights-advisor-frontend/blob/master/src/Utilities/TestingUtilities.js#L8) as well. That is why, I replaced TestWrapper with ComponentWithContext. Let's proceed with the review of this PR, I will, first of all, push the component into FEC so that we can share it among apps, and then adopt it in the vuln app in the next PR.

Reply:
Yes, I agree that `TestWrapper` is the better name for the component rather than `ComponentWithContext `. While pushing to FEC, I will keep this in mind.